CHANGES.rst

```diff
@@ -0,0 +1,7 @@
+Changelog
+=========
+
+0.6.2 (unreleased)
+----------------
+
+- Add support for PyPy, Python 2.7, and Python 3.
```

Comment:
There's a changelog in [src/zc/zodbdgc/README.txt](https://github.com/zopefoundation/zc.zodbdgc/blob/master/src/zc/zodbdgc/README.txt#L72).  I think the changelog shouldn't be split into two files.

My personal preference would be to have the entire changelog in CHANGES.rst at the top level, i.e. I think the existing changelog ought to be moved there.  But I'm not the primary zc.zodbdgc maintainer (in fact I didn't even know about its existence before I got a GitHub email about a new pull request in the zopefoundation org ;).

BTW if the changelog is moved, [setup.py](https://github.com/zopefoundation/zc.zodbdgc/blob/master/setup.py#L31) ought to be updated to concatenate it to the end of the README, so the PyPI page continues to display the changelog like it does now.